openapi/openapi.yaml

```
@@ -478,6 +478,9 @@ tags:
       Use these operations to manage password resets.
       When a password reset is requested, an email with a token is sent.
       In the email the user must click a reset link, and then create a new password.
+  - name: Risk Score
```

issue: we must use sentence case
```suggestion
  - name: Risk score
```